public/styles.css

```diff
@@ -3,12 +3,16 @@ header {
 	margin-bottom: 25px;
 }
 
+.container {
+	width: 585px;
+}
+
 header .container {
 	display: flex;
 	align-items: center;
 	max-width: 100%;
-	/* 1170 is bootstrap's widest breakpoint, and the 85 and 20 are for the logo. */
-	width: calc( 1170px + ( ( 85px + 20px ) * 2 ) );
+	/* 585 is the container width set above, and the 85 and 20 are for the logo. */
+	width: calc( 585px + ( ( 85px + 20px ) * 2 ) );
```

Comment:
This an above should maybe be `max-width` to retain the responsive design. Otherwise content exceeds the viewport on mobile devices.